### DISCUSSION TITLE
Install in vm without internet

### DISCUSSION THREAD

**Discussion by BloomingFlowers** [aquasecurity/trivy#5676] [Documentation]
### Description

I'm an ops of about 1k vms.
I've succed installing trivy in vm with the rpm,and here comes my problem.
How should I run trivy without internet,since it  must download DB at first time?
I've read the document about installation and found no answer.
Very appreciate to your help.
![5](https://github.com/aquasecurity/trivy/assets/22815724/756f8245-45f4-412f-ae28-301a722fb1d3)

### Suggestions

hope to add the installations without internet

**Comment by itaysk** [comment]
https://aquasecurity.github.io/trivy/v0.47/docs/advanced/air-gap/

**Reply by BloomingFlowers** [reply]
I do appreciate your kindness.
